### ISSUE TITLE
Shift+VolumeUp/Down? | Keyboard shortcuts settings location

### ISSUE THREAD

**Issue by trueTrueman** [ytmdesktop/ytmdesktop#827]
- [x] I understand that **YTMDesktop have NO affiliation with Google or YouTube**.
- [x] I verified that there is no open issue for the same subject.

I wanted to set the keyboard shortcuts for Increasing and Decreasing volume to the combinations of Shift/Ctrl/Alt + media buttons, so I could use the media buttons for both system volume and YTMDesktop volume with separate combinations, but that doesn't seem to work. I can successfully set just AudioVolumeUp and AudioVolumeDown, without Shift/Ctrl/Alt, but if I try to enter the combination of let's say Shift+AudioVolumeUp, nothing happens. Is it possible to use the combination and not just single buttons AudioVolumeUp/Down?

Maybe as a workaround it's possible to type in the keys manually? May I ask where the keyboard shortcut settings are stored?

**Environment (please complete the following information):**
 * YTMDesktop version: ***v1.13.0***
 * OS: ***Windows***
 * OS version: ***7***
 * Arch: ***x64***
* Installation way: ***.exe***

**Comment by Alipoodle** [comment]
On Windows 10 I was successfully able to set it to any Combination of CTRL/Shift/Alt + Media Key, however this did also trigger Windows to Update the Volume (Windows Media Service is on)
![image](https://user-images.githubusercontent.com/17199186/134810216-15c560e2-c841-4d7d-a994-45857e945150.png)

# Force Setting it
If you want to Force this outside of YTM to potentially work (although may have the issue of triggering both Windows & YTM)
Go to: `%appdata%/youtube-music-desktop-app` and Locate the `config.json` file.
Open this in Any Text Editor (although Notepad++ or Visual Studio Code will make it easier)

Near the top you should see something like this (although with your keybinds):
```json
        "settings-accelerators": {
		"media-play-pause": "disabled",
		"media-track-next": "disabled",
		"media-track-previous": "disabled",
		"media-track-like": "disabled",
		"media-track-dislike": "disabled",
		"media-volume-up": "CmdOrCtrl+Alt+Q",
		"media-volume-down": "CmdOrCtrl+Alt+A",
		"miniplayer-open-close": "disabled"
	},
```

Edit the Media Volume up & Down repectively With:
```"media-volume-up": "CmdOrCtrl+Alt+Shift+AudioVolumeUp",``` (This applies all 3 keys)
```"media-volume-down": "Shift+AudioVolumeDown",``` (Only applies to Shift Key)

**Comment by trueTrueman** [comment]
Thank you for your response! I wrote Shift+AudioVolumeDown and Shift+AudioVolumeUp into the config.json file and that appeared in YTMDesktop but when I pressed the combination, still nothing happened. The system volume also didn't change during that, while pressing single media buttons (without Shift) changed system volume succesfully.
Maybe it's something to do with Windows 7. Thank you for your response again.
